src/ServerClient/ADXClient.ts

```diff
@@ -1,4 +1,6 @@
 import { ADXResponse, RawADXResponse } from "./ADXResponse";
+const packageJson = require('../../package.json');
```

Comment:
[Looks like this has security implications ](https://stackoverflow.com/questions/9153571/is-there-a-way-to-get-version-from-package-json-in-nodejs-code)as all internal versions get exposed to the client

A readup metions something can be done at the [build stage](https://www.npmjs.com/package/genversion) and be used in the code. Something to look at may be..

Reply:
implemented through webpack